Show settings after install

bastilian opened the pull request:
This PR fixes #650 by not removing the settings button and show after a package has been installed that has settings.

For this I had to remove `_.memoize` for `PackageManager#packageHasSettings` as it returned a wrong result, this may effect rendering of longer package lists, but so far I have not noticed any negative effect.

lee-dohm commented:
There is a bug in this implementation. When I'm on a package's settings page, the settings button is displayed:

<img width="952" alt="screen_shot_2016-06-21_at_8_43_26_am" src="https://cloud.githubusercontent.com/assets/1038121/16236189/50294aa4-378c-11e6-93bc-1124e38cca45.png">

bastilian commented:
@lee-dohm I've added a spec to verify this and fixed it. The settings button in now shown after install and not shown on the settings page of a package.

lee-dohm commented:
Thanks very much @bastilian!
